packages/web-comment/src/utils/getPostcodeSearchResults.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+const { appealsApi } = require('../server.config');
+
+module.exports = {
```

Comment:
I think a better approach is an `AppealsApiClient` class, which can take the baseUrl as a constructor parameter, then have methods for specific endpoints. Thoughts?

Reply:
Yeah that's definitely a cleaner solution. Implemented that